etc/servo_automation_screenshot.py

```diff
@@ -0,0 +1,120 @@
+Here is the breakdown of how our code works:
+*   A session is started on localhost:7002
+*   The randomly generated webpage's (html files) data is sent as JSON to this session
+*   Using curl request, we load the html files for this session ID based on the session we just created.
+"""
+import os
+# import subprocess
+import json
+import requests
+import start_servo
+import time
+import base64
+import sys
+import getopt  # for command line interaction
+# import socket
+
+
+port = ''
+resolution = ''
+url_cl = ''
+file_url = ''
+
+
+def main(argv):  # take inputs from command line by considering the options parameter i.e -h, -p etc.
+    try:
+        # input options defined here.
+        opts, args = getopt.getopt(argv, "hu:p:i:r:", ["url=", "port=", "ifile=", "resolution="])
+    except getopt.GetoptError:
+        # an error is generated if the options provided in commandline are wrong.
+        # The help statement is printed on how to input command line arguments.
+        print('python3 etc/servo_automation_screenshot.py -u <url> -p <port> -i <html_file_url> -r <resolution>')
+        sys.exit(2)
+    for opt, arg in opts:
+        if opt == '-h':  # -h means help. Displays how to input command line arguments
+            print('python etc/servo_automation_screenshot.py -u <url> -p <port> -i <html_file_url> -r <resolution>')
+            sys.exit()
+        elif opt in ("-p", "--port"):  # store the value provided with the option -p in port variable.
+            global port
+            port = arg
+        elif opt in ("-u", "--url"):  # store the value provided with the option -u in url_cl variable.
+            global url_cl
+            url_cl = arg
+        elif opt in ("-i", "--ifile"):  # store the value provided with the option -i in inputfile variable.
+            global file_url
+            file_url = arg
+        elif opt in ("-r", "--resolution"):  # store the value provided with the option -i in inputfile variable.
+            global resolution
+            resolution = arg
+
+# This is to verify the name of the file is correct.
+# This is because the name of the file is the first argument passed through the command line.
+# After it is true it takes the command line arguments and processes it using the function main()
+if __name__ == "__main__":
+    if len(sys.argv) < 2:
+        print('Argument input is required in the format')
+        print('python3 etc/servo_automation_screenshot.py -u <url> -p <port> -i <html_file_url> -r <resolution>')
+        sys.exit()
+    else:
+        main(sys.argv[1:])
+
+
+# The below function is used to start servo on localhost:7002
+process_servo = start_servo.start_servo(port, resolution)
+
+# Since servo takes time to load and to get the session ID for subsequent steps, we added a sleep for 60 seconds
+time.sleep(60)
+# We are waiting for reponse and we will edit as soon as we get the response.
+
+# sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
As discussed by email, this is the right way to check for the webdriver server's availability.